main/EditText.tsx

```diff
@@ -287,18 +287,14 @@ const Component: FC<EditTextProps & {theme: DoobooTheme}> = ({
           <Text
             style={[
               compositeStyles.labelText,
-              styles?.labelText,
-              hovered && editable
-                ? [compositeStyles.labelTextHovered, styles?.labelTextHovered]
-                : editable
-                ? {
-                    color: errorText
-                      ? errorColor
-                      : focused
-                      ? focusColor
-                      : disableColor,
-                  }
-                : {color: disableColor},
+              hovered && editable && compositeStyles.labelTextHovered,
+              !editable
+                ? {color: disableColor}
+                : errorText
+                ? {color: errorColor}
+                : focused
+                ? {color: focusColor}
+                : {},
```

Comment:
Actually, the intention of this statement was to set the `hovered` color more prior than the `focusColor`. If not it would show a bit awkward on the web. To explain this more clearly, I've recorded the `gif`.

![hover](https://user-images.githubusercontent.com/27461460/125880607-ffdc5c36-4923-4112-985b-13e9a8a05724.gif)

If this is not understandable please tell me.

Reply:
In addition, the `focusColor` here is blue.